24.healthchecks/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.6
```

Comment:
Latest version for Alpine is 3.7. I don't expect that to be an issue, but we might want to use the latest available versions when we add new examples.